go.mod

```diff
@@ -12,4 +12,7 @@ require (
 )
 
 // Version has been removed from GitHub
-retract v1.19.0
+retract (
+	v1.20.0
+	v1.19.0
+)
```

Comment:
we should also retract v1.0.0 (see https://pkg.go.dev/go.einride.tech/lcm?tab=versions)

Reply:
Great catch!